v0.7.1 (#103)
dlstreamer/pipeline-server: docs/defining_pipelines.md

#### More Information

For more information and examples of media analytics pipelines created

> with Intel(R) DL Streamer please see the Intel(R) DL Streamer [tutorial](https://github.com/opencv/gst-video-analytics/wiki/GStreamer%20Video%20Analytics%20Tutorial).

Reviewer: ```suggestion
with Intel(R) DL Streamer please see the Intel(R) DL Streamer [tutorial](https://github.com/dlstreamer/dlstreamer/wiki/DL-Streamer-Tutorial).
```

## Model-Proc Files

In addition to the `.xml` and `.bin` files that are part of a model's
`IR` format, `Intel(R) DL Streamer` elements and `FFmpeg Video Analytics`
filters make use of an additional JSON file specifying the input and
output processing instructions for a model. Processing instructions
include details such as the expected color format and resolution of
the input as well labels to associate with a models outputs.
The Pipeline Server automatically looks for this file in the path
`models/model-alias/model-version/*.json`. Note that the model manager will
fail to load if there are multiple ".json" model-proc files in this directory.

### Intel(R) DL Streamer
For more information on Intel(R) DL Streamer `model-proc` files and samples for
common models please see the Intel(R) DL Streamer

> [documentation](https://github.com/opencv/gst-video-analytics/wiki/Model-preparation)

Reviewer: ```suggestion
[documentation](https://github.com/dlstreamer/dlstreamer/wiki/Model-preparation)
```

and

> [samples](https://github.com/opencv/gst-video-analytics/tree/master/samples/model_proc).

Reviewer: ```suggestion
[samples](https://github.com/dlstreamer/dlstreamer/tree/master/samples/model_proc).
```
